internal/portforward/service.go

```diff
@@ -129,8 +132,6 @@ type Service struct {
 	state    States
 }
 
-var _ PortForwarder = (*Service)(nil)
```

Comment:
Add this back in. This is used a compile time check to ensure that the struct satisfies the interface.

Reply:
Spotted another one we'd removed, which appears to be the same pattern, `var _ error = (*notFound)(nil)` which I've added back in.